torchtitan/experiments/llama4/model/args.py

```diff
@@ -35,9 +35,18 @@ class TransformerModelArgs(BaseModelArgs):
     depth_init: bool = True
     norm_type: str = "rmsnorm"
 
-    use_flex_attn: bool = False
-    attn_mask_type: str = "causal"
+    use_flex_attn: bool = True
+    attn_mask_type: str = "block_causal"
     eos_id: int = 0
+    # iRoPE settings
+    # When ``every_n_layers_nope`` is specified, NoPE (no positional embedding) is
+    # used every n layers. Other layers uses RoPE (rotary positional embedding) and
+    # the inner attention of those layer will use the fixed block size specified by
+    # ``fixed_attn_block_size``. ``fixed_attn_block_size`` means that the query will
+    # only attend to the tokens within the same block regardless how long is the
+    # sequence.
+    every_n_layers_nope: Optional[int] = None
```

Comment:
```suggestion
    every_n_layers_nope: int | None = None
```
and maybe remove `Optional` in this file